experimental/packages/opentelemetry-instrumentation/package.json

```diff
@@ -76,6 +76,7 @@
     "@opentelemetry/api": "^1.3.0"
   },
   "devDependencies": {
+    "@opentelemetry/sdk-metrics": "^1.8.0",
```

Comment:
I believe this needs to follow our dev dependency conventions to set a specific version instead of version ranges. However, I can not edit the PR before merging.

```suggestion
    "@opentelemetry/sdk-metrics": "1.8.0",
```

Reply:
Tips: when you create the PR, if you are ok with it, it would be more convenient for us to adopt those miscellaneous snippets before merging if you can check this option. 

![image](https://user-images.githubusercontent.com/8500303/201250543-f4762b3a-710e-461c-a339-a28b37d93cc1.png)

Reply:
Thanks! it will be enabled from my next pr (currently I'm under my company fork so I can't set this option).

Reply:
@osherv could you do the change and rebase so we can merge this ?

Reply:
ready for merge :)

Reply:
I still didn't find the caret being removed. Would you mind verifying if your commits are pushed?

Reply:
Has this been updated?